web-client/src/views/CreatePetitionerAccount/CreatePetitionerAccountForm.tsx

```diff
@@ -30,6 +35,18 @@ export const CreatePetitionerAccountForm = connect(
   }) => {
     const [inFocusEmail, setInFocusEmail] = useState(true);
     const [inFocusName, setInFocusName] = useState(true);
+    const [submitDisabled, setSubmitDisabled] = useState(false);
+
+    // Re-enabled submit button if submission was unsuccessful
+    useEffect(() => {
+      if (alertError || alertWarning) {
+        setSubmitDisabled(false);
+      }
+    }, [alertError, alertWarning]);
+
+    const submitFunction = debounce(() => {
+      submitCreatePetitionerAccountFormSequence();
+    }, 500);
```

Comment:
Just curious - how'd we land on 500ms for this? If I understand correctly, this is the amount of time that we'll wait before issuing the click? Is this delay noticeable to end users?

Reply:
Good question. My research online (read: Google/StackOverflow) suggested that 500 ms is a good starting point (see, e.g., https://ux.stackexchange.com/questions/95336/how-long-should-the-debounce-timeout-be), if on the high end; I chose a higher value to make sure there is virtually no chance to trigger the error. We can probably reduce this if need be.

That said, the user _does_ get "immediate" feedback: the button deactivates as soon as they click. So the 500 ms delay is lumped into the apparent network submission time, if that makes sense, so it shouldn't be too noticeable.

Reply:
My only nit is a suggestion to replace the magic number w/ a constant e.g. DEBOUNCE_TIME_MS or somesuch.